scene/gui/color_picker.h

```diff
@@ -68,13 +83,15 @@ class ColorPicker : public BoxContainer {
 	Size2i ms;
 	bool text_is_constructor = false;
 	int presets_per_row = 0;
+	PickerShapeType picker_type;
```

Comment:
We initialize the members in the .h file now. Also, I think the wheel shape should be made the default, as this is likely the most useful shape IMO.